src/org/zaproxy/zap/extension/exportreport/Utility/SharedFunctions.java

```diff
@@ -0,0 +1,210 @@
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.imageio.ImageIO;
+import javax.swing.BorderFactory;
+import javax.swing.ImageIcon;
+import javax.swing.JButton;
+import javax.swing.JComboBox;
+import javax.swing.JFormattedTextField;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTextArea;
+import javax.swing.JTextField;
+import javax.swing.SpringLayout;
+import javax.swing.UIManager;
+
+import org.apache.log4j.Logger;
+import org.parosproxy.paros.Constant;
+import org.zaproxy.zap.extension.exportreport.Export.ExportReport;
+
+/*
+ * AUTHOR		:	GORAN SARENKAPA - JordanGS
+ * SPONSOR		:	RYERSON UNIVERSITY
+ */
+
+public class SharedFunctions {
+	private static Logger logger = Logger.getLogger(ExportReport.class);
+
+	private static String DEFAULT_FONT = "";
+	private static boolean EXISTS_FONT = false;
+
+	public final static String DATE_FORMAT = "EEE, d MMM yyyy HH:mm:ss z(Z)";
+
+	public static String getCurrentTimeStamp() {
+		SimpleDateFormat date = new SimpleDateFormat(DATE_FORMAT);
+		Date now = new Date();
+		String strDate = date.format(now);
+		// System.out.println(strDate);
+		return strDate;
+	}
+
+	public static boolean fontExists(String name) {
+		GraphicsEnvironment g = null;
+		g = GraphicsEnvironment.getLocalGraphicsEnvironment();
+		String[] fonts = g.getAvailableFontFamilyNames();
+		for (int i = 0; i < fonts.length; i++) {
+			//System.out.println(fonts[i]);
+			if (fonts[i].equals(name)) {
+				setDefaultFont(name);
+				setExistsFont(true);
+				return true;
+			}
+		}
+		return false;
+	}
+
+	public static void setDefaultFont(String name) {
+		DEFAULT_FONT = name;
+	}
+
+	public static void setExistsFont(boolean bool) {
+		EXISTS_FONT = bool;
+	}
+
+	public static String getDefaultFont() {
+		return DEFAULT_FONT;
+	}
+
+	public static boolean getExistsFont() {
+		return EXISTS_FONT;
+	}
+	
+	public static Font getTitleFont() {
+		Map<TextAttribute, Integer> fontAttributes = new HashMap<TextAttribute, Integer>();
+		fontAttributes.put(TextAttribute.UNDERLINE, TextAttribute.UNDERLINE_ON);
+		String name = UIManager.getDefaults().getFont("Label.font").getFamily();
+		if (getExistsFont())
+			name = getDefaultFont();
+		//System.out.println(name);
+		return new Font(name, Font.BOLD, 30).deriveFont(fontAttributes);
+	}
+
+	public static Font getLabelFont() {
+		String name = UIManager.getDefaults().getFont("Label.font").getFamily();
+		if (getExistsFont())
+			name = getDefaultFont();
+		//System.out.println(name);
+		return new Font(name, Font.BOLD, 14);
+	}
+
+	public static void allignLabelTop(JLabel lbl) {
+		lbl.setVerticalAlignment(JLabel.TOP);
+		lbl.setVerticalTextPosition(JLabel.TOP);
+		lbl.setBorder(BorderFactory.createEmptyBorder(3 /* top */, 0, 0, 0));
+	}
+
+	public static JLabel createLabel(JPanel pnl, JLabel lbl, String str, Font font)
```

Comment:
Well, technically it's not using the same variable (the one passed is not the same being assigned).
You could just declare a new variable in the method and remove the parameter, i.e.:
```Java
JLabel lbl = new JLabel(str, JLabel.TRAILING);
...
```